db/migrate/20180803130417_add_column_to_calendar_events_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToCalendarEventsTable < ActiveRecord::Migration[5.1]
+  def change
+    add_column :calendar_events, :import_id, :bigint
```

Comment:
We should probably add an index on `import_id` otherwise query performance can deteriorate when the table starts getting large especially since we're going to be sorting by `import_id`.